doc/getting_started/_index.md

```diff
@@ -50,78 +46,88 @@ You can then **skip to step 4** (building software).
 If you do have Linux, you are still welcome to try the Docker container.
 However, as the container option is currently experimental, we recommend following the steps below to build manually if you plan on being a long-term user or contributor for the project.
 
-Our continuous integration setup runs on Ubuntu 18.04 LTS, which gives us the most confidence that this distribution works out of the box.
+Our continuous integration setup runs on Ubuntu 20.04 LTS, which gives us the most confidence that this distribution works out of the box.
 We do our best to support other distributions, but cannot guarantee they can be used "out of the box" and might require updates of packages.
 Please file a [GitHub issue](https://github.com/lowRISC/opentitan/issues) if you need help or would like to propose a change to increase compatibility with other distributions.
 
+You will need at least 7GB of RAM in order to build the Verilator simulation.
+If you have an FPGA, this constraint does not apply.
```

Comment:
Unless you are building the bitstream, then there is probably a memory constraint (from recent experiments we know 4GB is not enough,  though @dmcardle will probably have better numbers here). If you are just downloading a bitstream from our cloud bucket then obviously the mem limit doesn't apply.

Reply:
I'll rephrase this. 7GB is what our CI has, so it should be enough for both.

Reply:
nit:  Is it ok to say "at least 7GB of *available* RAM"?

Reply:
I'm not sure about the memory requirement for Verilator builds, actually. The thing I learned recently is that we need RAM >16 GB to build CW310 bitstreams with Vivado.

Reply:
Ouch. I got the 7GB from the fact that our CI runners have 7GB, and the person I was helping install who ran out of RAM said they kept running out until they increased their VM's allowance from 6GB to 8GB. So > 6GB, <= 7GB are the bounds we have for Verilator.

The updated comment here doesn't speculate about how much memory is needed for building the bitstream, since the default setup is not to build it locally and I don't want to put too much detail about custom setups in the "getting started" page.